### DISCUSSION TITLE
New Print/PDF Option for a Map (if possible)

### DISCUSSION THREAD

**Discussion by burnuser** [freeplane/freeplane#436] [Ideas]
I use Maps also for project documentation and reporting, which needs printable output.
The most beautiful Export option for me is PDF - showing the original map. But printing a bigger PDF Map gives a bad result on regular paper formats like A4 (=> resulting downscaled Text is to small on one page to read.)

I tried all Print and Export Options, but for me, the only good working (but a bit time consuming) solution for a really well formatted outline view is a multiple step procedure:

1.) Export Map as XHTML (JavaScript version)
2.) Open exported html file in Webbrowser
3.) Print from Webbrowser: Save as PDF => perfectly formatted on A4 (or whatever you want), with good looking line and page breaks.
4.) Delete no longer needed XHTML version

Any chance for a 1 click solution with similar results for Print/PDF?
(Only a question. I can live with my 4 step solution, if 1 click integration is not possible or too complicated.)

**Comment by dpolivaev** [comment]
select outline view and print to pdf printer

**Comment by burnuser** [comment]
Yes, is possible. But the line breaks of this solution are not responsive and the page breaks (cutting vertically and horizontally through the nodes) are in my opinion not usable at all. (And the layout is not very impressive.)

But it's OK, As stated, I can live with my 4 steps to get a really good looking result with responsive line breaks and working page breaks.
